microsetta_interface/implementation.py

```diff
@@ -3381,6 +3444,22 @@ def post_campaign_edit(body):
     return get_campaign_edit(campaign_info['campaign_id'])
 
 
+def get_account_removal_requests():
+    if not session.get(ADMIN_MODE_KEY, False):
+        raise Unauthorized()
+
```

Comment:
I believe this should be included as well
```suggestion
    account_details = session.get(LOGIN_INFO_KEY)
    if account_details is None:
        raise Unauthorized()

```

Reply:
I thought we had a decorator to manage these boilerplate items, but I may be misremembering